easybuild/easyconfigs/c/CheckM/CheckM-1.0.18-foss-2019a-Python-2.7.15.eb

```diff
@@ -0,0 +1,45 @@
+# Updated from previous config
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'PythonPackage'
+
+name = 'CheckM'
+version = '1.0.18'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/Ecogenomics/CheckM'
+description = """CheckM provides a set of tools for assessing the quality of genomes recovered from isolates,
+ single cells, or metagenomes."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+source_urls = ['https://pypi.python.org/packages/source/c/checkm-genome']
+sources = ['checkm-genome-%(version)s.tar.gz']
+checksums = ['0dcf31eab5e340a0fff37d7a5091d46d9269b0708db8f789adcd7cbd2a09a2b7']
+
+options = {"modulename": "checkm"}
+
+dependencies = [
+    ('Python', '2.7.15'),
+    ('pplacer', '1.1.alpha19', '', True),
+    ('prodigal', '2.6.3'),
+    ('HMMER', '3.2.1'),
+    ('SciPy-bundle', '2019.03'),  # numpy
+    ('matplotlib', '2.2.4', versionsuffix),
+    ('Pysam', '0.15.2'),
+    ('DendroPy', '4.4.0'),
+]
+
+download_dep_fail = True
+
+use_pip = True
+
+tests = ["%(installdir)s/bin/checkm test ~/checkm_test_results"]
```

Comment:
This is storing the test results in the home directory, which is not ideal.

Let's use the top-level build directory instead?

```python
tests = ["%(installdir)s/bin/checkm test %(builddir)s/checkm_test_results"]
```